src/chrome/content/rules/Cp.pt.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Cp.pt">
```

Comment:
Mismatch:
https://www2.cp.pt/
https://www1.cp.pt/
https://ns0.cp.pt/

Redirect:
http://venda.cp.pt

Invalid cert:
https://cloud.cp.pt/

Timeout:
https://sites.cp.pt/
http://amf.cp.pt/


You should add:
https://cpkids.cp.pt/
https://portal.cp.pt/
https://webmail.cp.pt/